src/app/api/boatInfo/[id]/route.ts

```diff
@@ -0,0 +1,31 @@
+import { Client } from "@notionhq/client";
+
+const notionDatabaseId: string | undefined = process.env.NOTION_DATABASE_ID;
+const notionSecret: string | undefined = process.env.NOTION_SECRET;
```

Comment:
Undefined is a `nullable` value fot the objects, for the string we can  `process.env.NOTION_SECRET || ''`